lib/cisco_node_utils/cmd_ref/README_YAML.md

```diff
@@ -469,6 +472,23 @@ By convention, a `default_value` of `''` (empty string) represents a configurabl
 
 `config_get()` will return the defined `default_value` if the defined `config_get_token` does not match anything on the node. Normally this is desirable behavior, but you can use [`auto_default`](#auto_default) to change this behavior if needed.
 
+### `default_only`
+
+Some attributes may be hard-coded on certain platforms in such a way that they have a meaningful `default_value` but no relevant `config_get_token` or `config_set` behavior. On such platforms, the key `default_only: true` can be set, which causes the `config_get()` API to always return the default value and `config_set()` to raise a `Cisco::UnsupportedError`.
```

Comment:
Good thought. I can *maybe* see a case where setting it explicitly to `false` is desirable (say it's default_only for all platforms with *one* exception?) and I think it'd be confusing to have `default_only: false` be ignored and have the flag still set to true.

However, I can certainly change this one so that `default_only:` implies `default_only: true` - and make the same change to allow `multiple:` while I'm at it. Sound reasonable?